src/test/pmemset_source/pmemset_source.c

```diff
@@ -436,6 +436,246 @@ test_src_from_file_with_truncate(const struct test_case *tc, int argc,
 	return 1;
 }
 
+/*
+ * test_src_from_file_with_rusr_mode - test source creation with
+ * PMEMSET_SOURCE_FILE_CREATE_ALWAYS_MODE flags.
+ */
+static int
+test_src_from_file_with_rusr_mode(const struct test_case *tc, int argc,
+		char *argv[])
+{
+	if (argc < 1)
+		UT_FATAL("usage: test_src_from_file_with_rusr_mode <path>");
+
+	const char *file = argv[0];
+	struct pmemset_source *src;
+	uint64_t flags = 0;
+	int ret = 0;
+
+	flags = PMEMSET_SOURCE_FILE_CREATE_ALWAYS | \
+	PMEMSET_SOURCE_FILE_CREATE_ALWAYS_MODE(PMEMSET_SOURCE_FILE_RUSR_MODE);
```

Comment:
Is there a reason why file_create_always flag is used if there's also file_create_always_mode flag used?